### PR TITLE
View Part

### PR DESCRIPTION
**Pull request by SysOverdrive**
All the UTags are showned in a STree and now the UI can be very easily modified if something is not in order.

Search is just for show.

Removed helper code.

### CODE REVIEW

File: Source/UTagsEd/Private/SUtagsListWidget.cpp [hunk 1]
```diff
@@ -0,0 +1,167 @@
+
+#include "SUtagsListWidget.h"
```
Comment (andreihaidu) [inline Source/UTagsEd/Private/SUtagsListWidget.cpp]: SUtagsListWidget -> SUTagsListWidget

File: Source/UTagsEd/Private/SUtagsTreeViewWidget.cpp [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+	ItemCounterIndex = 0;
+
+	// UI Implementation
+	ChildSlot[
+		SNew(SBox)
+			//	.MaxDesiredHeight(ChildSlot) FOR SCROLLBAR WE MUST GET THE SIZE OF THE CHILDREN AND SET IT TO THE MAXIMUM OF THE WINDOW
+			[
+				SNew(SBorder)[
+					SNew(SScrollBox)
+						+ SScrollBox::Slot()[//TODO Aici putem pune border color
```
Comment (andreihaidu) [inline Source/UTagsEd/Private/SUtagsTreeViewWidget.cpp]: English comments pls

File: Source/UTagsEd/Private/UTagsEdModeToolkit.cpp [hunk 3]
```diff
@@ -0,0 +1,104 @@
+// Copyright 1998-2018 Epic Games, Inc. All Rights Reserved.
```
Comment (andreihaidu) [inline Source/UTagsEd/Private/UTagsEdModeToolkit.cpp:1]: Uni bremen copyright, and you as author (look at the other examples I have)

File: Source/UTagsEd/Private/desktop.ini [hunk 4]
```diff
@@ -0,0 +1,4 @@
+[LocalizedFileNames]
```
Comment (andreihaidu) [inline Source/UTagsEd/Private/desktop.ini]: What is desktop.ini? Is it required, does it have to be in /Source, not /Config?

File: Source/UTagsEd/Public/SUtagsListWidget.h [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+#include "SlateBasics.h"
+#include "SSearchBox.h"
+
+#include "UTags/Public/UTags.h"
+#include "UTags/Public/Tags.h"
+#include "LevelEditor.h"
+#include "Runtime/Engine/Classes/Engine/Engine.h"
+
+class SUtagsListWidget : public SCompoundWidget
```
Comment (andreihaidu) [inline Source/UTagsEd/Public/SUtagsListWidget.h]: tags -> Tags

File: Source/UTagsEd/UTagsEd.Build.cs [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+			);
+			
+		
+		PublicDependencyModuleNames.AddRange(
+			new string[]
+			{
+				"Core",
+                "UTags",
+                "UIds",
+                "UConversions",
```
Comment (andreihaidu) [inline Source/UTagsEd/UTagsEd.Build.cs]: UConversions is not required

File: Source/UTagsEd/Private/SUtagsListWidget.cpp (hunk 1 above)
Comment (SysOverdrive) [reply Source/UTagsEd/Private/SUtagsListWidget.cpp]: This is actually not present in the last version of the code.

**Review by SysOverdrive — COMMENTED** [review]
This file was deleted from the last commit

**Comment by SysOverdrive** [comment]
Took notice of the changes and prepared a new Pull Request.

**Pull request closed without merging**